### ISSUE TITLE
Please add TinyMediaManager and Audiobookshelf

### ISSUE THREAD

**Issue by tboucher204** [NX211/homer-icons#79]
![audiobookshelf](https://user-images.githubusercontent.com/17454305/228036501-66607b45-33d2-4198-a877-bfaa773488ae.png)
![tmm](https://user-images.githubusercontent.com/17454305/228036543-5c170835-d082-4a2d-8c69-869ef2422e55.png)

I have added a couple images for your consideration.

**Comment by petestanley** [comment]
Yep would say both are must have's for this community. @tboucher204 maybe log a PR to make adding a 2-click process for the maintainer.
